xml/SkinSettings.xml

```diff
@@ -77,6 +77,12 @@
 					<onclick>Skin.ToggleSetting(no_fanart)</onclick>
 					<selected>!Skin.HasSetting(no_fanart)</selected>
 				</control>
+				<control type="radiobutton" id="606">
+					<label>$LOCALIZE[31163]</label>
+					<include>DefaultSettingButton</include>
+					<onclick>Skin.ToggleSetting(hide_background_fanart)</onclick>
+					<selected>!Skin.HasSetting(hide_background_fanart)</selected>
+				</control>
 				<control type="button" id="607">
```

Comment:
the panel container from dialog 1105 would need to get converted to a grouplist. (example: Custom_1101_SettingsList.xml)